sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -42,13 +44,16 @@ enum class PollSocketDirection
  * @param socketFileDescriptor socket descriptor.
  * @param direction poll events for read or write socket.
  * @param timeout  return if polling for more than \p timeout
+ * @param context The context while polling that can be use to cancell waiting for socket.
+ *
  * @return int with negative 1 upon any error, 0 on timeout or greater than zero if events were
  * detected (socket ready to be written/read)
  */
 int pollSocketUntilEventOrTimeout(
```

Comment:
Context argument should be first parameter.  pollSocketUntilEventOrTimeout is a method and should be PollScoket.....